app/views/layouts/application.html.erb

```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
   <title>ShareChristmas</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
```

Comment:
do we want to move this to the bottom of `body`? I'm not sure how much JS we'll be writing so I don't know if that would be an issue